appveyor.yml

```diff
@@ -1,14 +1,12 @@
 environment:
   LINUX_IMAGE: &linux_image Ubuntu2204
-  MACOS_IMAGE: &macos_image macos-catalina
+  MACOS_IMAGE: &macos_image macos-monterey
   WINDOWS_IMAGE: &windows_image Visual Studio 2022
   LIBZEN_VERSION: 0.4.41
   MEDIAINFO_VERSION: 23.10
   UBUNTU_VERSION: 22.04
   QA_PYTHON_VERSION: 3.12
-  # Python 3.12 is not available everywhere yet, see https://github.com/appveyor/ci/issues/3879
```

Comment:
Can you leave this comment but update it to mention 3.13 and https://github.com/appveyor/ci/issues/3927 please?

Reply:
Makes sense. I had to update the MacOS image to a newer version, `macos-catalina` does not support py3.12.